tfx/components/transform/executor_test.py

```diff
@@ -129,7 +132,7 @@ def _verify_transform_outputs(self, materialize=True):
         tf.saved_model.SAVED_MODEL_FILENAME_PB)
     self.assertTrue(tf.io.gfile.exists(path_to_saved_model))
 
-  def _runPipelineGetMetrics(self, inputs, outputs, exec_properties):
+  def _run_pipeline_get_metrics(self):
```

Comment:
Most of this file is camelCase, so I suggest switching those that are not to camelCase instead for consistency.

Reply:
The reason why I updated the name for this was because it originally had a pylint error -- invalid-name.

I don't know if we should switch those to camel case because I checked other executor_test files and I found out that helper functions are usually like this (_verify_test) and only tests are camel case (testDo). You think it's ok to leave it like this and be consistent with other files?

Reply:
Yes, I can do this, so to confirm, I will change all the method names to snake_case, including the tests?

Reply:
Hi Zohar, I updated all the tft tests to snake_case.

Reply:
Yes, thank you.